src/static/riot/tasks/management.tag

```diff
@@ -28,6 +30,8 @@
         <tbody>
         <tr each="{ task in tasks }" onclick="{show_detail_modal.bind(this, task)}" class="task-row">
             <td>{ task.name }</td>
+            <td class="benchmark-row">{ task.competitions.join(', ') }</td>
+            <td>{ task.shared_with.join(', ') }</td>
```

Comment:
When you change from using method fields to nested serializer you will have to change this to something like.
```suggestion
            <td class="benchmark-row">{ task.competitions.title }, { task.competitions.id }</td>
            <td>{ task.shared_with.name }</td>
```
You could even add a link to the user's profile if you think it fits.